server/src/main/java/org/jetbrains/bsp/bazel/server/sync/ProjectFileParser.kt

```diff
@@ -0,0 +1,50 @@
+package org.jetbrains.bsp.bazel.server.sync
+
+import com.google.protobuf.TextFormat
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.toList
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.bsp.bazel.info.BspTargetInfo
+import java.net.URI
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.attribute.FileTime
+import kotlin.io.path.getLastModifiedTime
+
+class ProjectFileParser {
```

Comment:
maybe TargetInfoReader. ProjectFile can be a confusing name, as we have 4 project models in bazell-bsp (from aspect, common, bsp and bloop).

Reply:
Renamed